app/src/main/java/ch/blinkenlights/android/vanilla/MediaSessionTracker.java

```diff
@@ -124,6 +112,9 @@ public void updateSession(Song song, int state) {
 			boolean showCover = SharedPrefHelper.getSettings(mContext).getBoolean(PrefKeys.COVER_ON_LOCKSCREEN, PrefDefaults.COVER_ON_LOCKSCREEN);
 			if (showCover) {
 				metadataBuilder.putBitmap(MediaMetadataCompat.METADATA_KEY_ALBUM_ART, cover);
+				// Checking this for debug purposes
+				// I suspect Vanilla fails at rendering album art timely
+				mMediaSession.setMetadata(metadataBuilder.build());
```

Comment:
Can this be removed? I don't see why this is needed since we will call the same function on line 134 anyway later